#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()
gen.add("tolerance", double_t, 0, "margin length of each clustering", 0.02, 0.0, 1.0)
gen.add("label_tracking_tolerance", double_t, 0, "margin length of label tracking", 0.2, 0.0, 1.0)
gen.add("max_size", int_t, 0, "the max number of the points of each cluster", 25000, 0, 50000)
gen.add("min_size", int_t, 0, "the minimum number of the points of each cluster", 20, 0, 1000)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "EuclideanClustering"))
